src/ci/bin/testCentaurHoricromtalPapiV2beta.sh

```diff
@@ -26,8 +26,9 @@ cromwell::build::run_centaur \
     -e standard_output_paths_colliding_prevented \
     -e papi_v2alpha1_gcsa \
     -e restart \
-    -e lots_of_inputs \
-    # `lots_of_inputs` is redundant to, and a less rigorous version of, `lots_of_inputs_papiv2`
+    -e lots_of_inputs_papiv2 \
+    # Due to Centaur changes in WX-1629, lots_of_inputs_papiv2 times out due to the large number of inputs. Instead,
+    # we run lots_of_inputs, which tests 400 inputs instead of the 10,000 in lots_of_inputs_papiv2
```

Comment:
Aside from this test – did the slower Centaur evaluation inflate the overall suite time? I can't immediately intuit whether it would matter.

Reply:
Comparing the length of time of the tests on this PR to the nightly integration tests, the timings are about the same (some suites are a little faster, some are a little slower, all are pretty close though). I think that one test was problematic because it creates an array of 10,000 Files, which causes a recursion stack of 10,000 calls when we process the array. No other tests create such large arrays, and I think that call stack might have caused memory issues and slowness (I didn't test this hypothesis, but it would make sense since that seems to be the main difference between this test and others).